docs/Makefile

```diff
@@ -1,42 +1,161 @@
-SPHINXOPTS    ?=
-SPHINXBUILD   ?= sphinx-build
-SOURCEDIR     = .
-BUILDDIR      = _build
-LINKCHECKDIR  = _build/html
+# Makefile for Sphinx documentation
```

Comment:
I had made this commit during the pre-proposal phase, so it would be hard to double down on a single source just yet. It would probably be better to exclude [da35aca](https://github.com/koordinates/kart/pull/659/commits/da35aca24bdbf647fbced4aadecf757a0590e5e0) in this PR since it bore more relevance with the sphinx-click integration.

Reply:
ah, just looked like boilerplate — does it come from sphinx?

Reply:
I don't think so. The sphinx boilerplate is smaller.